controldev.orogen

```diff
@@ -74,6 +74,42 @@ task_context "GenericRawToMotion2D" do
     port_driven :raw_command
 end
 
+# This task converts controldev/RawCommand to base/Twist
+# with configurable axes, ranges and deadzones
+task_context "GenericRawToTwist" do
```

Comment:
- Use comments for documentation (just put the doc as a comment on top of the property/port)
- Let's maybe try to reduce the number of properties ... Could rename `max_*` into `scale_*` and then one can set a negative scale to get an inversion (no code change apart from the removal of `invert_`)
- use `static inline` functions instead of macros. No need to set the target within the function, just return the value and set it from the updateHook

Reply:
The task was mainly copied from GenericRawToMotion2D (https://github.com/rock-drivers/drivers-orogen-controldev/blob/master/controldev.orogen#L55), including the documentation and property names, I kept the naming scheme for the properties instead of introducing new ones (for a similar task).

Using a static inline function would need to provide all used properties and parameters in the call, currently they are generated by the macro:
_##AXISNAME.get()
_##AXISNAME##_deadzone
_invert_##AXISNAME
_max_##AXISNAME

When adding more parameters to the function call, the code would imo become more confusing and more copy/paste error prone. But I don't have strong feelings about that.